databricks/koalas/groupby.py

```diff
@@ -640,6 +641,54 @@ def diff(self, periods=1):
             lambda sg: sg._kser._diff(periods, part_cols=sg._groupkeys_scols)
         )
 
+    def cumcount(self, ascending=True):
+        """
+        Cumulative count for each group.
+
+        Returns
+        -------
+        Series or DataFrame
```

Comment:
Would you mind matching those docstring to pandas ?

you can print the whole docstring using `help(pd.core.groupby.GroupBy.cumcount)`


```python
"""
    Number each item in each group from 0 to the length of that group - 1.

    Essentially this is equivalent to

    >>> self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))

    Parameters
    ----------
    ascending : bool, default True
        If False, number in reverse, from length of group - 1 to 0.

    Returns
    -------
    Series
        Sequence number of each element within each group.
    ...
"""
```

Reply:
Sure, can do so. I was more aligning with the docstring from `cummax`, but the one from pandas is also fine